cpu/stm32f7/ldscripts/stm32f746zg.ld

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup      cpu_stm32l4
+ * @{
+ *
+ * @file
+ * @brief           Memory definitions for the STM32F746ZG
+ *
+ * @author          Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+MEMORY
+{
+    rom (rx)    : ORIGIN = 0x08000000, LENGTH = 1024K
+    ram (rwx)   : ORIGIN = 0x20000000, LENGTH = 320K
+    cpuid (r)   : ORIGIN = 0x1ff80050, LENGTH = 12
```

Comment:
this should be `0x1ff0f420` see [datasheet](http://www.st.com/content/ccc/resource/technical/document/reference_manual/c5/cf/ef/52/c0/f1/4b/fa/DM00124865.pdf/files/DM00124865.pdf/jcr:content/translations/en.DM00124865.pdf) section 41.1 page 1654